github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -1048,3 +1048,28 @@ postsubmits:
                 memory: "8Gi"
               limits:
                 memory: "8Gi"
+    - name: mirror-images-from-docker-hub
+      always_run: false
+      run_if_changed: "hack/mirror-images.sh"
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      cluster: ibm-prow-jobs
+      labels:
+        preset-kubevirtci-quay-credential: "true"
```

Comment:
Given that we are going to deal with docker hub it would be good to include docker proxy settings `preset-docker-mirror-proxy: "true"`

Reply:
Added.